### PR TITLE
Query params don't have to declared in route path

### PR DESCRIPTION
**Pull request by devinrhode2**
I definitely got caught up on this going from one exercise to the next, there's probably a better way to explain the distinction between query params and other params.. I'm not even exactly sure of the distinction yet. (But a PR happens to be an easy and great way to communicate my suggestion, and yeah it can be directly and easily merged ;)

### CODE REVIEW

**Comment by devinrhode2** [comment]
To further clarify, the mistake I had that was stuck on was having the route's path set to `'/{name}'`, to match the query param `'?name=Handling'`

**Comment by lloydbenson** [comment]
In hapi ? syntax isn't supported (see the hapijs module called call if you are interested).  So in this case /{name} in the route path would translate to: localhost:port/Handling and then it would say Handling in your view.  I think it could likely be more clear but I don't think the proposal you have actually helps that. With this information, do you have another proposal that you think would help clarify it a bit more?

**Comment by lloydbenson** [comment]
Sorry responded too quickly here.  I had forgotten about the query option.  (I never used it and I wrongly recalled this exercise was params not query).  Investigating this a bit more.

File: exercises/views/problem.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ surrounding the variable with curly braces, e.g. `{{foo}}`.
 
 The template receives some information from the request. For example, the query
 parameters that were passed in via the URL are available in the `query` object.
-These parameters can then be used in the template.
+These parameters can then be used in the template. Query params don't have to be declared in the route `path`.
```
Comment (lloydbenson) [inline exercises/views/problem.md]: Maybe:

Query params get automatically parsed and aren't declared in the route path.

Comment (lloydbenson) [inline exercises/views/problem.md]: If that seems fine feel free to change it (or counter propose and I'll get this merged in).

**Comment by lloydbenson** [comment]
I didn't hear from you so I did a PR myself.  I also noticed that the solution is wrong and fixed.  See: https://github.com/hapijs/makemehapi/pull/116.

**Pull request closed without merging**